nil/getting-started/working-with-smart-contracts/deploying-a-contract.mdx

```diff
@@ -0,0 +1,20 @@
+# Deploying a contract
+
+## External deployment
+
+An external deployment of a contract is made by a user or another entity outside of the cluster. 
+
+As is the case with all external messages, an external deployment must be paid by the contract at the address to which the call is maid. However, before deployment, this address is unoccupied. To proceed with external deployment, an entity must first send funds to the address where the contract will reside.
+
+:::info
+
+Once funds are sent to an empty address, the address is 'locked'. Only the owner of the funds can deploy a contract to this address.
```

Comment:
I'm not sure we should describe it this way. 

A better way would be to say that the address is calculated in a way that only the owner of a specific keypair can deploy a wallet there. So it's safe to send funds to this address. There is no actual "locking" taking place.

Also, as soon as money ends up on the address, it is occupied, but uninitialized.